.github/workflows/parsetree-change.yml

```diff
@@ -0,0 +1,37 @@
+name: Parsetree Updated
+
+on:
+  pull_request:
+    paths:
+      - 'parsing/parsetree.mli'
+      - 'parsing/asttypes.mli'
+      - 'parsing/location.mli'
+      - 'parsing/longident.mli'
+      - 'parsing/parse.mli'
+      - 'parsing/pprintast.mli'
+
+jobs:
+  comment-and-label:
+    if: ${{! contains(github.event.pull_request.labels.*.name, 'parsetree-change')}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify ppxlib maintainers
+        uses: actions/github-script@v7
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'CC @NathanReb @patricoferris'
```

Comment:
I think it would be a better to create a ppxlib-maintainer team in the `ocaml-ppx` organization and CC that team rather than a fixed set of people.